src/ui/authenticationInApp/ViewAuthenticationSignUp.qml

```diff
@@ -29,6 +36,9 @@ MZInAppAuthenticationBase {
     _inputs: MZInAppAuthenticationInputs {
         objectName: "authSignUp"
 
+        _telemetryScreenId: authSignUp._telemetryScreenId
```

Comment:
I'm surprised this is necessary. Is `_telemetryScreenId` not inherited from `MZAuthenticationBase`?

Reply:
Eh, this could just be my QML ignorance showing really.

Reply:
I tried removing it but it is not inherited it seems like.